### PR TITLE
[Snyk] Fix for 2 vulnerabilities

### PR DESCRIPTION
**Pull request by khkh-ms**
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br />snyk-top-banner

#### Snyk changed the following file(s):

- `src/Azure.Functions.Cli/Azure.Functions.Cli.csproj`

#### Vulnerabilities that will be fixed with an upgrade:

|  | Issue | Score | 
:-------------------------:|:-------------------------|:-------------------------
medium severity | Concurrent Execution using Shared Resource with Improper Synchronization (&#x27;Race Condition&#x27;) <br/>[SNYK-DOTNET-AZUREIDENTITY-7246762](https://snyk.io/vuln/SNYK-DOTNET-AZUREIDENTITY-7246762) | &nbsp;&nbsp;**561**&nbsp;&nbsp; 
medium severity | Concurrent Execution using Shared Resource with Improper Synchronization (&#x27;Race Condition&#x27;) <br/>[SNYK-DOTNET-MICROSOFTIDENTITYCLIENT-7246763](https://snyk.io/vuln/SNYK-DOTNET-MICROSOFTIDENTITYCLIENT-7246763) | &nbsp;&nbsp;**561**&nbsp;&nbsp; 

---

> [!IMPORTANT]
>
> - Check the changes in this PR to ensure they won't cause issues with your project.
> - Max score is 1000. Note that the real score may have changed since the PR was raised.
> - This PR was automatically created by Snyk using the credentials of a real user.

---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: 
🧐 [View latest project report](https://app.snyk.io/org/khkh-ms/project/d5d421aa-b4b9-4ee9-b2c6-e843a7b78590?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)
📜 [Customise PR templates](https://docs.snyk.io/scan-using-snyk/pull-requests/snyk-fix-pull-or-merge-requests/customize-pr-templates)
🛠 [Adjust project settings](https://app.snyk.io/org/khkh-ms/project/d5d421aa-b4b9-4ee9-b2c6-e843a7b78590?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)
📚 [Read about Snyk's upgrade logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

---

**Learn how to fix vulnerabilities with free interactive lessons:**

🦉 [Learn about vulnerability in an interactive lesson of Snyk Learn.](https://learn.snyk.io/?loc&#x3D;fix-pr)

[//]: # 'snyk:metadata:{"customTemplate":{"variablesUsed":[],"fieldsUsed":[]},"dependencies":[{"name":"Azure.Identity","from":"1.11.2","to":"1.11.4"},{"name":"Microsoft.Identity.Client","from":"4.60.3","to":"4.61.3"}],"env":"prod","issuesToFix":[{"exploit_maturity":"No Known Exploit","id":"SNYK-DOTNET-AZUREIDENTITY-7246762","priority_score":561,"priority_score_factors":[{"type":"freshness","label":true,"score":71},{"type":"fixability","label":true,"score":214},{"type":"cvssScore","label":"5.5","score":275},{"type":"scoreVersion","label":"v1","score":1}],"severity":"medium","title":"Concurrent Execution using Shared Resource with Improper Synchronization ('Race Condition')"},{"exploit_maturity":"No Known Exploit","id":"SNYK-DOTNET-MICROSOFTIDENTITYCLIENT-7246763","priority_score":561,"priority_score_factors":[{"type":"freshness","label":true,"score":71},{"type":"fixability","label":true,"score":214},{"type":"cvssScore","label":"5.5","score":275},{"type":"scoreVersion","label":"v1","score":1}],"severity":"medium","title":"Concurrent Execution using Shared Resource with Improper Synchronization ('Race Condition')"}],"prId":"ea1937e9-55d0-4e13-a6ff-f66a22d7d8df","prPublicId":"ea1937e9-55d0-4e13-a6ff-f66a22d7d8df","packageManager":"nuget","priorityScoreList":[561,561],"projectPublicId":"d5d421aa-b4b9-4ee9-b2c6-e843a7b78590","projectUrl":"https://app.snyk.io/org/khkh-ms/project/d5d421aa-b4b9-4ee9-b2c6-e843a7b78590?utm_source=github&utm_medium=referral&page=fix-pr","prType":"fix","templateFieldSources":{"branchName":"default","commitMessage":"default","description":"default","title":"default"},"templateVariants":["priorityScore"],"type":"auto","upgrade":["SNYK-DOTNET-AZUREIDENTITY-7246762","SNYK-DOTNET-MICROSOFTIDENTITYCLIENT-7246763"],"vulns":["SNYK-DOTNET-AZUREIDENTITY-7246762","SNYK-DOTNET-MICROSOFTIDENTITYCLIENT-7246763"],"patch":[],"isBreakingChange":false,"remediationStrategy":"vuln"}'

### CODE REVIEW

**Comment by jviau** [comment]
@khkh-ms there is a conflict - can you update this PR?

**Comment by mattchenderson** [comment]
These changes are already present in the 4.x branch. This PR is no longer needed. I'll comment on the lines with relevant links before closing.

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 1]
```diff
@@ -294,7 +294,7 @@
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.834.3" />
     <PackageReference Include="Microsoft.Build" Version="17.0.0" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.60.3" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.61.3" />
```
Comment (mattchenderson) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:297]: Already present. See https://github.com/Azure/azure-functions-core-tools/blob/2b7cdb580f3197b97312222a7dc9b304c8e4d9f0/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj#L297

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 2]
```diff
@@ -284,7 +284,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.6.2" />
-    <PackageReference Include="Azure.Identity" Version="1.11.2" />
+    <PackageReference Include="Azure.Identity" Version="1.11.4" />
```
Comment (mattchenderson) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:287]: Already present. See https://github.com/Azure/azure-functions-core-tools/blob/2b7cdb580f3197b97312222a7dc9b304c8e4d9f0/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj#L287

**Pull request closed without merging**